CMakeLists.txt

```diff
@@ -71,7 +71,7 @@
 #    DEV: static includes blosc.a and blosc.h
 
 
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 3.1) # Threads::Threads target available from 3.1
```

Comment:
This is quite a strong requirement for a library to tries to be compatible with pretty old OS distros.  I suggest to use the [c-blosc2](https://github.com/Blosc/c-blosc2) project for such a innovative patches.  Thanks!

Reply:
@FrancescAlted Thank you for the feedback, I completely understand. I will close this PR in this case, it already got too heavy.

The requirement might be lowered if not for the Threads target but I don't think its worth the hassle.